pkg/apiserver/statement/service.go

```diff
@@ -113,9 +113,9 @@ func (s *Service) modifyConfigHandler(c *gin.Context) {
 	db := utils.GetTiDBConnection(c)
 
 	if !config.Enable {
-		err = db.Exec(buildConfigUpdateSQL(&config, "Enable")).Error
+		err = db.Exec(buildConfigUpdateSQL(&config, "Enable"), &config).Error
 	} else {
-		err = db.Exec(buildConfigUpdateSQL(&config)).Error
+		err = db.Exec(buildConfigUpdateSQL(&config), &config).Error
```

Comment:
Maybe better to integrate `buildConfigUpdateSQL` and `db.Exec` together! I guess the implementation will be easier to understand.

Reply:
Right, let me try.